src/components/dashboard-widgets/dashboard-enum-widget/dashboard-enum-widget.component.html

```diff
@@ -0,0 +1,49 @@
+<ux-predefined-widget class="widget-element"
+                      [id]="id"
+                      [colSpan]="colSpan"
+                      [rowSpan]="rowSpan"
+                      [name]="name"
+                      [heading]="heading"
+                      [fixedMode]="fixedMode">
+
+    <ux-input-dropdown class="enum-dropdown"
```

Comment:
`ux-select` probably works better here. There's a lot of spurious code here (like the pipe) that can be eliminated by using existing functionality.

Reply:
We wanted to display icons in the drop-down an I am not sure how we can achieve that with `ux-select`.

Reply:
You can use `optionTemplate`, e.g. https://plnkr.co/edit/8yQzzjak0u3guZY3

Reply:
And how do I display the icon on the closed drop-down?

Reply:
What icon?

Reply:
The closed input component shows a text and an icon, like the list on the open drop-down.

Reply:
This icon is customizable:
![image](https://user-images.githubusercontent.com/26114101/106446739-8549ee00-6478-11eb-8b40-2a62739559f7.png)